openedx/core/djangoapps/schedules/models.py

```diff
@@ -5,10 +5,18 @@
 
 class Schedule(TimeStampedModel):
     enrollment = models.OneToOneField('student.CourseEnrollment', null=False)
-    active = models.BooleanField(default=True, help_text=_('Indicates if this schedule is actively used'))
-    start = models.DateTimeField(help_text=_('Date this schedule went into effect'))
+    active = models.BooleanField(
+        db_index=True,
```

Comment:
I suspect this won't be useful as a stand-alone index, particularly given the cardinality. Is there a usage pattern you had in mind that you think this would address?

Instead we probably want some compound indexes on (start, active) and (upgrade_deadline, active) etc.